src/test/regress/expected/schema_based_sharding.out

```diff
@@ -1675,6 +1675,11 @@ FROM public.citus_schemas WHERE schema_name::text LIKE 'citus\_sch_' ORDER BY sc
 
 \c - - - :master_port
 SET search_path TO regular_schema;
+-- test we handle create schema with authorization properly for distributed schema
+SET citus.enable_schema_based_sharding TO ON;
+CREATE ROLE a;
+CREATE SCHEMA AUTHORIZATION a;
+SET citus.enable_schema_based_sharding TO OFF;
```

Comment:
would be nice to add a check to confirm that the schema is actually distributed

also, should drop the role at the end